ascii_output_v3.py

```diff
@@ -0,0 +1,100 @@
+"""
+Turns csv output of tax calculator to ascii output with uniform columns
+and transposes data so columns are rows, rows are columns
+
+"""
+
+#records the data of one csv row to be later written into a uniform column
+#also returns number of rows in result file
+def rec_col(data, col_size, temp):
+    nrows = 0
+    while (data != []):
+        col_elem = data.pop()
+        #cuts a data string from the end if length is larger than col_size
+        if (len(col_elem) > col_size):
+            cut = col_size - len(col_elem)
+            col_elem = col_elem[:cut]
+        else:
+        #adds spaces to fill out data string if length is smaller than col_size
+            col_elem = col_elem.ljust(col_size)
```

Comment:
For this kind of work (formatting a column to match up some size), I would suggest "formatters" in DataFrames:
http://stackoverflow.com/questions/20937538/how-to-display-pandas-dataframe-using-a-format-string-for-columns